lib/src/tag/_specs.dart

```diff
@@ -160,11 +166,13 @@ class Platform {
     'Linux',
     Runtime.broadNative,
     tag: 'platform:linux',
+    allowsNativeJit: true,
```

Comment:
Unclear why we need this hack?

Maybe, `broadNative` should include `nativeJit`?

Or is that going to get the wrong detection in some cases?

I suppose cases that works on web and native dart, would get detected as supported as working under flutter web? and there for get `sdk:flutter` and `sdk:dart` with `platform:linux` even if it uses mirrors on linux (which won't work with flutter).

I'm not sure I see a mechanism to solve that. Maybe, we need to detect sdk and then use that to detect platforms... or use the set of supported platforms to limit the set of supported sdks, saying that if it can't have an SDK tag if it doesn't satisfy all detect platforms for the sdk.